src/reanimated2/Hooks.js

```diff
@@ -16,16 +16,13 @@ import { getTag } from './NativeMethods';
 import NativeReanimated from './NativeReanimated';
 import { Platform } from 'react-native';
 
-export function useSharedValue(init, shouldRebuild = true) {
+export function useSharedValue(init) {
   const ref = useRef(null);
   if (ref.current === null) {
     ref.current = {
       mutable: makeMutable(init),
       last: init,
```

Comment:
I don't think `last` is needed. We can perhaps just do: `ref.cuurent = makeMutable(init)`

Reply:
fixed a4e7f6e